packages/s3/lib/streams.js

```diff
@@ -34,3 +34,63 @@ export const createWritableStream = (path) => {
   const stream = createWriteStream(path)
   return toWritableStream(stream)
 }
+
+/**
+ * Convert a ReadableStream to a Readable.
+ *
+ * @param {any} webStream The WebStream to convert.
+ * @returns {Readable} The Node.js stream.
+ */
+export const toReadable = (webStream) => {
+  const nodeStream = new Readable({
+    // The read method is called when the stream wants to pull more data in
+    read: () => { },
+  })
+
+  // Handle each chunk of data
+  const reader = webStream.getReader()
+  const pump = () => {
+    reader.read().then(({ done, value }) => {
+      if (done) {
+        // When no more data, close the stream
+        nodeStream.push(null)
+      } else {
+        // Push the data into the Node.js stream
+        nodeStream.push(Buffer.from(value))
+        pump()
+      }
+    }).catch(err => {
+      // Handle any errors
+      nodeStream.emit('error', err)
+    })
+  }
+
+  // Start pumping data
+  pump()
+
+  return nodeStream
+}
+
+/**
+ * Get a string out of a Readable.
+ *
+ * @param {Readable} stream Stream to read from.
+ * @returns {Promise<string>} String.
+ */
+export const toString = async (stream) => {
+  const data = []
+
+  return new Promise((resolve, reject) => {
+    stream.on('data', (chunk) => {
+      data.push(chunk)
+    })
+
+    stream.on('end', () => {
+      resolve(data.join(''))
+    })
+
+    stream.on('error', (error) => {
+      reject(error)
+    })
+  })
+}
```

Comment:
Why not use `get-stream` package?

Reply:
As it's only used for tests and that it's a simple function, I don't think it's required to add extra packages ; but I can add it.

Reply:
It is already wildly used in other packages also for tests

Reply:
Done!